source/extensions/transport_sockets/alts/BUILD

```diff
@@ -0,0 +1,86 @@
+licenses(["notice"])  # Apache 2
```

Comment:
I think we're generally doing docs links in the build files, which is a good reminder to add docs on what this is and how to configure it :-)